web-app/src/app/tabs/faults/faults.component.html

```diff
@@ -1,5 +1,130 @@
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.fanMonitorFault == true, 'off' : batteryFaults.errorFlags.fanMonitorFault == false}">Fan Monitor Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.highVoltageIsolationFault == true, 'off' : batteryFaults.errorFlags.highVoltageIsolationFault == false}">High Voltage Isolation Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.internalCommunicationFault == true, 'off' : batteryFaults.errorFlags.internalCommunicationFault == false}">Internal Communications Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.internalConversionFault == true, 'off' : batteryFaults.errorFlags.internalConversionFault == false}">Internal Conversion Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags. internalLogicFault == true, 'off' : batteryFaults.errorFlags.internalLogicFault == false}">Internal Logic Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.internalMemoryFault == true, 'off' : batteryFaults.errorFlags.internalMemoryFault == false}">Internal Memory Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.internalThermistorFault == true, 'off' : batteryFaults.errorFlags.internalThermistorFault == false}">Internal Thermistor Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.lowCellVoltageFault == true, 'off' : batteryFaults.errorFlags.lowCellVoltageFault == false}">Low Cell Voltage Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.openWiringFault == true, 'off' : batteryFaults.errorFlags.openWiringFault == false}">Open Wiring Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.packVoltageSensorFault == true, 'off' : batteryFaults.errorFlags.packVoltageSensorFault == false}">Pack Voltage Sensor Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.thermistorFault == true, 'off' : batteryFaults.errorFlags.thermistorFault == false}">Thermistor Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.twelvevpowerSupplyFault == true, 'off' : batteryFaults.errorFlags.twelvevpowerSupplyFault == false}">Twelve V Power Supply Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.voltageRedundancyFault == true, 'off' : batteryFaults.errorFlags.voltageRedundancyFault == false}">Voltage Redundancy Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.weakCellFault == true, 'off' : batteryFaults.errorFlags.weakCellFault == false}">Weak Cell Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.weakPackFault == true, 'off' : batteryFaults.errorFlags.weakPackFault == false}">Weak Pack Fault</div>
```

Comment:
I would also try to make this smaller so that it fits the side panels either by adding a margin to the whole faults component or making 2 columns for battery faults but up to you.

Reply:
Eventually, the faults page will be redesigned so it acts like the faults page in the debug display (Only active faults appear) so I'll just leave this for now.